src/main/java/org/ua/javaPro/berezhnoy/hillelHomeWork11/ArrayInitializer.java

```diff
@@ -0,0 +1,38 @@
+package org.ua.javaPro.berezhnoy.hillelHomeWork11;
+
+public class ArrayInitializer {
+
+    public static void init(double[] array) {
+        int halfLength = array.length / 2;
+        Thread thread1 = new Thread(new Runnable() {
+            @Override
+            public void run() {
+                for (int i = 0; i < halfLength; i++) {
+                    array[i] = array[halfLength + i] = formula(i, array[i]);
```

Comment:
```suggestion
                    array[i] =  formula(i, array[i]);
```
інакше другий потік робить те саме.